cleo/light/light.py

```diff
@@ -294,6 +294,8 @@ class Light(Stimulator):
     wavelength: Quantity = field(default=473 * nmeter, kw_only=True)
     """light wavelength with unit (usually nmeter)"""
 
+    scan_freq : int = field(default=30, kw_only=True)
```

Comment:
I'd create a subclass RasterScanningLight to hide the scan_freq details from the typical Light object